spice_simulation_generator/component_sim.py

```diff
@@ -0,0 +1,33 @@
+# Creates spice deck to simulate a particular component
+
+import spice_gen
+
+# Top level 
+TOP = "bitcell_array"
+
+# Name of component in act file
+NAME = "bitcell_array<1,1>"
+
+# ACT file where everything is located (same directory)
+ACT = "bitcell.act"
+
+# IO pins for module
+PINS = ["wl",  "bl", "bl_"]
+#PINS = ["din[0]", "din[1]", "wl[0]", "wl[1]", "go", "sen", "wen", "pchg", "wreq", "Reset", "en1_s", "en2_s", "en1_m", "en2_m", "dout[0]", "dout[1]", "dr", "wc"]
+
+# Model file
+MODEL = "/gpfs/gibbs/pi/manohar/tech/SKY/130/skywater-pdk/libraries/sky130_fd_pr/latest/models/sky130.lib.spice"
```

Comment:
This is very specific to a particular install. Can we change this to $ACT_HOME/$ACT_TECH/models.sp?